utils/das_data_sorter.py

```diff
@@ -1,6 +1,79 @@
 import pandas as pd 
 import argparse 
 from math import ceil
+from numpy import median, mean
+
+# averages data within the seconds or minutes, according to new_time array
+def avg_Data(array):
```

Comment:
Was it intentional to have two functions called `avg_Data` ? It also looks like theres a small difference in both of them too, so which one did you want?

Also forgot to say this last time, can you rename this method to `average_data`. See https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables on how to name methods using the PEP 8 python conventions.